### PR TITLE
Prepare for next release

### LINKED ISSUE #3: Add Binder with example Notebook
- [ ] add `introduction.ipynb` to this repository with examples of main features
- [ ] add install script so that GDAL is available, see https://repo2docker.readthedocs.io/en/latest/config_files.html
- [ ] works on MyBinder.org
- [ ] add Binder badge

### LINKED ISSUE #60: Add Introduction notebooks

### LINKED ISSUE #66: Library raises warning msg of missing CRS when `time interval` is extracted.
Screenshot from 2020-04-07 13-14-04

### PR DESCRIPTION
**Pull request by YouQam**
closes #3: Add Binder with example Notebook
closes #60: Add Introduction notebooks 
closes #66: Library raises warning msg of missing CRS when `time interval` is extracted.

### CODE REVIEW

**Comment by nuest** [comment]
@YouQam Why did you remove GDAL ?

**Comment by YouQam** [comment]
> @YouQam Why did you remove GDAL ?

From where? Can you elaborate?

File: geoextent/lib/extent.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
+logging.disable(logging.INFO)
```
Comment (nuest) [inline geoextent/lib/extent.py:13]: I think this use of `logging.disable` is not what it is intended for. From [the function documentation](https://docs.python.org/3.5/library/logging.html#logging.disable):

>  `logging.disable(lvl)` Provides an overriding level lvl for all loggers which takes precedence over the logger’s own level. When the need arises to temporarily throttle logging output down across the whole application, this function can be useful. 

The key words being "temporarily" and "for the whole application". None of which is happening here.

We want to make the logging configurable for users > see https://github.com/o2r-project/geoextent/issues/68

Comment (YouQam) [reply geoextent/lib/extent.py:13]: Nice idea!

Comment (YouQam) [reply geoextent/lib/extent.py]: I enabled logging. https://github.com/YouQam/geoextent/commit/49e0a709c687bf57e500143574441202b0e28650

**Pull request merged**